src/app/user/modules/records/components/record-detail/record-detail.component.spec.ts

```diff
@@ -44,19 +35,11 @@ describe('RecordDetailComponent', () => {
     snackBarOpen = jasmine.createSpy();
     dialogRef = jasmine.createSpyObj('', ['afterClosed']);
     TestBed.configureTestingModule({
-      declarations: [ RecordDetailComponent, OffCampusRecordDetailComponent ],
+      declarations: [ RecordDetailComponent ],
       imports: [
-        MatCardModule,
-        MatIconModule,
-        MatPaginatorModule,
-        MatFormFieldModule,
-        MatSelectModule,
-        MatInputModule,
-        MatDividerModule,
-        MatDialogModule,
-        FormsModule,
         NoopAnimationsModule,
         HttpClientTestingModule,
+        SharedModule,
```

Comment:
I think you don't have to import `SharedModule`, instead, you can add the component you need in `declarations`.